using_images/xpaas_images/eap.adoc

```diff
@@ -9,7 +9,94 @@
 
 toc::[]
 
-If you'd like to contribute to OpenShift documentation, see our
-https://github.com/openshift/openshift-docs[source repository] and
-https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc[guidelines]
-to get started.
+== Overview
+Red Hat JBoss Enterprise Application Platform (JBoss EAP) is available as a containerized xPaaS image that is designed for use with OpenShift. Developers can quickly build, scale, and test applications deployed across hybrid environments.
+
+[IMPORTANT]
+There are significant differences in supported configurations and  functionality in the JBoss EAP xPaaS image compared to the regular release of JBoss EAP.
+
+This topic details the differences between the JBoss EAP xPaaS image and the regular release of JBoss EAP, and provides instructions specific to running and configuring the JBoss EAP xPaaS image. Documentation for other JBoss EAP functionality not specific to the JBoss EAP xPaaS image can be found in the  https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/[JBoss EAP documentation on the Red Hat Customer Portal].
+
+The location of `_EAP_HOME_` inside a JBoss EAP xPaaS image is: *_/opt/eap/_*. 
+
+== Differences Between the JBoss EAP xPaaS Image and the Regular Release of JBoss EAP
+
+=== Functionality Differences for OpenShift JBoss EAP xPaaS Images
+
+There are several major functionality differences in the OpenShift JBoss EAP xPaaS image:
+
+* The JBoss EAP Management Console is not available link:#Managing-OpenShift-JBoss-EAP-xPaaS-Images[to manage OpenShift JBoss EAP xPaaS images].
+* The JBoss EAP Management CLI is only bound locally. This means that you can only access the Management CLI of a container from within the pod. For example: 
++
+----
+oc exec p <pod> -c <container> - /opt/eap/bin/jboss-cli.sh
+----
++
+[WARNING]
+Any configuration changes made using the JBoss EAP Management CLI on a running container is not persistent. 
+* Domain mode is not supported. OpenShift controls the creation and distribution of applications in the containers.
+* There is no HornetQ messaging subsystem. The JBoss A-MQ xPaaS image can be used for scenarios that require messaging. 
+
+[[Managing-OpenShift-JBoss-EAP-xPaaS-Images]]
+=== Managing OpenShift JBoss EAP xPaaS Images
+
+A major difference in managing an OpenShift JBoss EAP xPaaS image is that there is no Management Console exposed for the JBoss EAP installation inside the image.
+
+As containers are not persistent, the JBoss EAP Management Console is not included. However, the JBoss EAP Management CLI (*_EAP_HOME/bin/jboss-cli.sh_*) is still accessible from within the container for troubleshooting purposes, however any configuration changes made on a running OpenShift container will be lost when the container is destroyed.
```

Comment:
on further reflection i think we need to elaborate on the "containers not persistent therefore no config" explanation.  It's more like "Because images are intended to be immutable, with modifications being written to a non-persistent filesystem, the Management Console is not exposed"